GeneralsMD/Code/GameEngine/Source/GameClient/MapUtil.cpp

```diff
@@ -721,7 +738,12 @@ Bool MapCache::addMap( AsciiString dirName, AsciiString fname, FileInfo *fileInf
 	{
 		DEBUG_LOG(("Missing TheKey_mapName!\n"));
 		AsciiString tempdisplayname;
-		tempdisplayname = fname.reverseFind('\\') + 1;
+		if(fname.reverseFind('\\')) {
+			tempdisplayname = fname.reverseFind('\\') + 1;
+		}
+		else if(fname.reverseFind('/')) {
+			tempdisplayname = fname.reverseFind('/') + 1;
+		}
 		md.m_displayName.translate(tempdisplayname);
```

Comment:
It looks like there's cases where this path can have either windows or linux, so to be safe, checking for both.